_posts/2016/2016-03-25-governance.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: Governance and open source infrastructure
+tags:
+- opensource
+---
+
+There has been a lot of discussions this week following the [NPM dispute](http://blog.npmjs.org/post/141577284765/kik-left-pad-and-npm) about what the community should/shouldn't be doing. While I agree with most of what has been said, I think there's a deeper issue that hasn't been explored here:
+
+_Open source infrastructure–like GitHub and Node–lacks an adequate model for open source governance._
+
+"Governance" is a fancy word for: who makes decisions, how are those decisions made, and how do actions based on those decisions get carried out? It is especially important when we're dealing with shared resources, like land, air, the crap we take out of or put into the land or air, and money. We are most familiar with the concept from our national governments.
+
+But "governance" can apply to things besides nation states. My childhood home had a two-party system that created and enforced the policies that governed our land, money, and citizens. This system was ruled by my mom who weilded most of the power, including the ability to unilaterally create new policies, enforce law and order, rule on disputes amongst the citizens, and imprison anyone on a whim. My dad technically had veto power over everything, but he rarely exercised it due to the fear of political repercussions. None of this was written in law books, but there was an inherent governance system.
+
+Governance is especially important when shared resources are subject to rivalry–what economists call _rivalrous_. If I can interfere with your use of a resource, then it is rivalrous. There are many ways governance deals with rivalrous resources, such as charing you to use them, penalizing you for abusing them, or assigning governance of specific things to others.
```

Comment:
tautological first sentence. maybe 'resources cannot be enjoyed simultaneously by multiple parties', or 'resources are depleted with each use', or ' resources have a marginal production cost greater than zero'...

Second sentence is not quite right. If my use of a resources can interfere with your use... My ability to interfere just says I have power, not necessarily anything about the resource.